datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -136,10 +137,11 @@ def start(ctx, check, env, agent, python, dev, base, env_vars):
     echo_success('success!')
 
     echo_waiting('Starting the Agent... ', nl=False)
-    result = environment.start_agent()
-    if result.code:
+    try:
+        environment.start_agent()
+    except docker.errors.ContainerError as e:
         click.echo()
-        echo_info(result.stdout + result.stderr)
```

Comment:
we could get split stdin and stdout by passing a flag but I don't really see an use for that